source/overview/license-and-subscription.rst

```diff
@@ -109,13 +109,80 @@ Is my license available immediately?
 
 Yes, once your payment is successfully processed your license is immediately available to download.
 
+How will I know when to renew my license?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You will be notified 60 days prior to your license expiry that your license is due for renewal, via a blue banner displayed at the top of your Mattermost window. This banner is only visible to System Admins.
+
+You can select **Please renew** to begin the renewal process. You can also select the **x** to dismiss the notification. The notification is reactivated when your browser refreshes.
+
+How do I renew my license?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can renew your license in Mattermost via the blue renewal reminder banner. Select **Please Renew** to begin the process. You can also visit https://mattermost.com/renew, and complete the form provided. 
+
+How long does it take to renew a license?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once you’ve started the renewal process, we will be in contact with you to confirm your order and send you the order form. There may be additional paperwork required. Once we have the signed order form and (if applicable) the necessary paperwork from you, we are able to process the renewal and issue your license key within 24 hours.  
+
+What happens to my license if I don't renew in time?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you don't renew within the 60-day renewal period, a 10 day grace period is provided for you to upload a new license key. During this period your Mattermost installation runs as normal, with full access to Enterprise features. 
+
+When the grace period expires, your Enterprise version is downgraded to Team Edition. Enterprise features are disabled.
+ 
+What happens when my license expires?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you don't renew your license within the 10-day grace period, your Mattermost version is automatically downgraded to Team Edition so you can still access and use Mattermost. However, Enterprise features will no longer be available and if you are currently using them, the functionality will no longer be accessible. 
+
+When you renew, the Enterprise features will become available with the previous configuration (provided no action such as user migration has been taken). 
+
+Which features are affected when my Enterprise license expires?
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+The affected Enterprise features include, but are not limited to, the following:
+
+.. csv-table::
+    :header: "Feature", "How it's affected", "What steps do I need to take?"
```

Comment:
I'm unsure if the items on the LHS of the table need periods.

Reply:
Good point. I've removed them, looks better. :)